spec/javascripts/modules/track-click.spec.js

```diff
@@ -24,6 +25,59 @@ describe('A click tracker', function() {
 
     element.trigger('click');
 
+    expect(GOVUK.analytics.setDimension).not.toHaveBeenCalled();
     expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith('category', 'action', {label: 'Foo'});
   });
+
+  it('tracks breadcrumb click events', function() {
+    spyOn(GOVUK.analytics, 'setDimension');
+    spyOn(GOVUK.analytics, 'trackEvent');
+
+    element = $(
+      '<a data-track-category="breadcrumbClicked" \
+          data-track-action="1" \
+          data-track-label="/" \
+          data-track-dimension="Home" \
+          data-track-custom-dimension="29" \
+          data-module="track-click" \
+          href="/">Home</a>'
+    );
+
+    tracker.start(element);
+
+    element.trigger('click');
+
+    expect(GOVUK.analytics.setDimension).toHaveBeenCalledWith(29, 'Home');
+    expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith(
+      'breadcrumbClicked',
+      1,
+      { label: '/' }
+    );
+  });
+
+  it('tracks related item click events', function() {
```

Comment:
Could you please also add a test that makes sure we call `GOVUK.analytics.trackEvent` but don't call `GOVUK.analytics.setDimension` when the custom dimension attribute isn't present?